playbooks/setup_telegraf_local.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Instal telegraf client on Localhost
+  hosts: local
+  become: true
+  vars_files:
+  - "./vars/telegraf_dbconnection_vars.yml"
+  - "./vars/nginx_telegraf_local.yml"
+  roles:
+  - role: telegraf-client
+    vars:
+      telegraf_plugins_base:
+        - name: http_listener_v2
+          options:
+            service_address: ":8080"
```

Comment:
Port should be moved to vars into `nginx_telegraf_local.yml` and be reused in `extra_parameters`